src/classes/os-utils.ts

```diff
@@ -57,6 +58,21 @@ export default class OsUtils {
     }
   }
 
+  public static detectOS = () : string => {
+    let detectedOS : string;
+    if(windows){
```

Comment:
Fix formatting (all ifs)
```suggestion
    if (windows) {
```